nodeup/pkg/model/context.go

```diff
@@ -110,55 +114,66 @@ func (c *NodeupModelContext) CNIBinDir() string {
 	}
 }
 
+// KubeletBootstrapConfig is the path the bootstrap config file
+func (c *NodeupModelContext) KubeletBootstrapConfig() string {
```

Comment:
Nit: KubeletBootstrapKubeconfig would keep it closer to the field name in the Kubelet API type